### PR TITLE
Fix: documentation examples on getting started for server

### PR DESCRIPTION
**Pull request by Aragur**
Fixes some breaking issues in the code samples of the Python SDK.  
This was done during Hacktoberfest 🎃 :)
Would be nice if this PR could get merged in time or get the required label.

### CODE REVIEW

File: app/views/docs/getting-started-for-server.phtml [hunk 1]
```diff
@@ -146,10 +146,11 @@ $client = (new Client())
         <div class="ide margin-top-small" data-lang="python" data-lang-label="Python SDK">
             <pre class="line-numbers"><code class="prism language-python" data-prism>from appwrite.client import Client
 
-client = Client()
+client = (Client()
     .set_endpoint('https://[HOSTNAME_OR_IP]/v1') # Your API Endpoint
     .set_project('5df5acd0d48c2')                # Your project ID
     .set_key('919c2db5d4...a2a3346ad2')          # Your secret API key
```
Comment (gewenyu99) [inline app/views/docs/getting-started-for-server.phtml:152]: Good catch!

I think I personally would prefer these to be formatted like this to  stay concise:
```py
client = (Client()
  .set_endpoint('https://[HOSTNAME_OR_IP]/v1') # Your API Endpoint
  .set_project('5df5acd0d48c2')                # Your project ID
  .set_key('919c2db5d4...a2a3346ad2'))         # Your secret API key
  ```

File: app/views/docs/getting-started-for-server.phtml [hunk 2]
```diff
@@ -415,10 +417,13 @@ $user = $users->create(
     <li>
         <h3>Python</h3>
         <div class="ide margin-top-small" data-lang="python" data-lang-label="Python SDK">
-            <pre class="line-numbers"><code class="prism language-python" data-prism>users = Users(client)
+            <pre class="line-numbers"><code class="prism language-python" data-prism>from appwrite.id import ID
```
Comment (gewenyu99) [inline app/views/docs/getting-started-for-server.phtml:420]: This example continues from previous examples and doesn't show the imports for brevity and continuity. All the other examples also don't show imports.

If you believe this section without imports seems confusing, consider how you would rephrase the sentences above so readers understand that this is the next step from previous examples.

Otherwise, consider reverting this change :)

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Hi there!

Thank you for your contribution. These a good catches, we really appreciate your effort.

I've left some of my thoughts here, please take a look at addressing them, then we can move to merge your changes.

Thanks!

File: app/views/docs/getting-started-for-server.phtml (hunk 2 above)
Comment (Aragur) [reply app/views/docs/getting-started-for-server.phtml:420]: So for me it was pretty confusing following this tutorial, because I didn't know where to import these files from.

File: app/views/docs/getting-started-for-server.phtml (hunk 1 above)
Comment (Aragur) [reply app/views/docs/getting-started-for-server.phtml:152]: Yes that definitely looks better, I will change it to reflect your proposal

Comment (Aragur) [reply app/views/docs/getting-started-for-server.phtml:152]: Yes that definitely looks better, I will change it to reflect your proposal

File: app/views/docs/getting-started-for-server.phtml (hunk 2 above)
Comment (Aragur) [reply app/views/docs/getting-started-for-server.phtml:420]: So for me it was pretty confusing following this tutorial, because I didn't know where to import these files from.

Comment (gewenyu99) [reply app/views/docs/getting-started-for-server.phtml:420]: > So for me it was pretty confusing following this tutorial, because I didn't know where to import these files from.

I recommend looking at the sections above this to see if there is a more natural way to lead the user to this section, if not, please update all examples in the docs here for consistency. 

Thanks ❤️

Comment (Aragur) [reply app/views/docs/getting-started-for-server.phtml]: I added a note.  
Maybe this is too much - if so, I think it's no problem to revert this change.  
Maybe I was the only stupid one to don't look down 4 rows to see the complete example :)

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: app/views/docs/getting-started-for-server.phtml [hunk 3]
```diff
@@ -377,6 +377,7 @@ let client = Client()
 <h2><a href="/docs/getting-started-for-server#makeRequest" id="makeRequest">Make Your First Request</a></h2>
 
 <p>After your SDK configuration is set, create any of the Appwrite service objects and choose any request to send. Full documentation for any service method you would like to use can be found in your SDK documentation or in the API References section.</p>
+<p>Note: To improve readability, we don't display the required imports in this example.</p>
```
[the remark this answers is not in the document]
Comment (Aragur) [reply app/views/docs/getting-started-for-server.phtml]: No Problem.
I added all the imports for all languages.

File: app/views/docs/getting-started-for-server.phtml [hunk 4]
```diff
@@ -382,7 +382,9 @@ let client = Client()
     <li>
         <h3>Node.js</h3>
         <div class="ide margin-top-small" data-lang="nodejs" data-lang-label="Node.js SDK">
-            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const users = new Users(client);
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const { Client, ID, Users } = require('node-appwrite');
```
Comment (gewenyu99) [inline app/views/docs/getting-started-for-server.phtml:385]: This is the last thing I'll ask for I swear. Let's remove the imports that are not used
```suggestion
            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const { ID, Users } = require('node-appwrite');
```

**Comment by Aragur** [comment]
Ping @gewenyu99
Any updates to this PR?

File: app/views/docs/getting-started-for-server.phtml [hunk 5]
```diff
@@ -401,7 +404,11 @@ promise.then(function (response) {
         <h3>PHP</h3>
 
         <div class="ide margin-top-small" data-lang="php" data-lang-label="PHP SDK">
-            <pre class="line-numbers"><code class="prism language-php" data-prism>$users = new Users($client);
+            <pre class="line-numbers"><code class="prism language-php" data-prism>use Appwrite\Client;
```
Comment (gewenyu99) [inline app/views/docs/getting-started-for-server.phtml:407]: I think you're still importing an unused `Client` class here and in other examples. Please make sure the imports are concise and consistent.

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: app/views/docs/getting-started-for-server.phtml (hunk 5 above)
Comment (Aragur) [reply app/views/docs/getting-started-for-server.phtml]: Sorry, I missed that completely.
It should now be removed everywhere.

**Review by gewenyu99 — APPROVED** [review]
LGTM! Thank you so much for your help <3

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)
